app/views/installations/header_tabs/providers/_github.html.haml

```diff
@@ -0,0 +1,6 @@
+- if @integration.selected_application.nil?
+  = form_tag integration_path(@integration), :method => 'put' do
+    = select_tag "integration[application]", grouped_options_for_select(@integration.get_applications), { class: "pull-left selectpicker" }
+    = submit_tag "Submit", { class: "pull-right col-sm-4 btn btn-default btn-primary"}
+- else
+  = 'Your app is ' + @integration.selected_application
```

Comment:
If you are doing special partials we should at least be correct and say `Your repository is ` instead of app.